From 8ab55c717d5088d8c75b465f5b9196036e0968ce Mon Sep 17 00:00:00 2001
From: Al Bailey <Al.Bailey@windriver.com>
Date: Wed, 3 Jan 2018 10:39:37 -0600
Subject: [PATCH] Remove puppetlabs-apt as ceph requirement

We will never install apt or puppet-apt, so this requirement cannot be fulfilled
---
 metadata.json | 1 -
 1 file changed, 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index 14d2f37..27b78e3 100644
--- a/metadata.json
+++ b/metadata.json
@@ -8,7 +8,6 @@
   "project_page": "https://launchpad.net/puppet-ceph",
   "issues_url": "https://bugs.launchpad.net/puppet-ceph",
   "dependencies": [
-    {"name":"puppetlabs/apt","version_requirement":">=2.0.0 <3.0.0"},
     {"name":"puppetlabs/apache","version_requirement":">=1.4.1 <2.0.0"},
     {"name":"puppetlabs/concat","version_requirement":">=1.2.1 <3.0.0"},
     {"name":"puppetlabs/inifile","version_requirement":">=1.0.0 <2.0.0"},
-- 
2.7.4

